Cargo.toml

```diff
@@ -11,12 +11,12 @@ categories = ["game-engines", "rendering"]
 resolver = "2"
 
 [dependencies]
-bevy = { version = "0.9", default-features = false, features = ["render"] }
+bevy = { version = "0.10.0", default-features = false, features = ["bevy_render"] }
 
-bevy_mod_raycast = "0.7"
+bevy_mod_raycast = {git = "https://github.com/soerenmeier/bevy_mod_raycast", branch = "bevy-0.10"}

```

Comment:
Temporarily pointing `bevy_mod_raycast` dependency to a 0.10-compatible fork. There is an active pull request to merge this branch into the main repo, so this will need to be updated once that happens.